src/main/java/uk/gov/hmcts/reform/laubackend/cases/service/CaseActionService.java

```diff
@@ -101,4 +104,17 @@ private Pageable getPage(final String size, final String page) {
 
         return of(parseInt(pageNumber) - 1, parseInt(pageSize), Sort.by(Sort.Direction.DESC, "timestamp"));
     }
+
+    public JurisdictionsCaseTypesResponse getJurisdictionsAndCaseTypes() {
+        List<JurisdictionCaseTypePair> pairs = caseActionAuditRepository.getJurisdictionsCaseTypes();
+        Set<String> jurisdictions = new HashSet<>();
+        Set<String> caseTypes = new HashSet<>();
+        pairs.forEach(pair -> {
+            jurisdictions.add(pair.getJurisdiction());
```

Comment:
Please trim jurisdictions and case types.

Reply:
that is a bit of a tricky one. If it had spaces around, searching would not work, unless provided with exact spaces around. So auto-suggest might work better with spaces, as they are in the DB.